src/styles/layouts/_main.scss

```diff
@@ -30,6 +32,7 @@
   max-width: $max-width-sidebar;
   display: flex;
   flex-direction: column;
+  z-index: 999;
```

Comment:
We should set some z-index in the settings, to control them in a better way. Don't do this in this PR, let's make another one unifying the z-index